rep-0003.rst

```diff
@@ -180,6 +179,52 @@ Buildsystem support:
 
   - build from source
 
+Kinetic Kame (May 2016 - May 2021)
+----------------------------------
+Required Support for:
+
+- Ubuntu Wily (15.10)
+- Ubuntu Xenial (16.04)
+
+Minimum Requirements:
+
+- C++11
+
+  - GCC 4.9 on Linux, as it's the version that Debian Jessie ships with
+
+- Python 2.7
+
+  - Python 3.4 not required, but testing against it is recommended
+
+- Lisp SBCL 1.2.4
+- CMake 3.0.2
+
+  - Debian Jessie ships with CMake 3.0.2
+
+- Boost 1.55
+
+  - Debian Jessie ships with Boost 1.55
```

Comment:
Are there no API differences between Boost 1.55 (debian) and 1.58 (ubuntu)? Just a bit concerned.